Shortly.Codes.BLL.Abstractions/Requests/RemoveCodeRequest.cs

```diff
@@ -0,0 +1,3 @@
+﻿namespace Shortly.Codes.BLL.Abstractions.Requests;
+
+public sealed record RemoveCodeRequest(string Email, string Value);
```

Comment:
```suggestion
public sealed record ManagerRemoveCodeRequest(string Email, string Value);
```
Чтобы название запросов от слоя к слою различались